app/models/manageiq/providers/cisco_intersight/physical_infra_manager.rb

```diff
@@ -8,6 +8,8 @@ class PhysicalInfraManager < ManageIQ::Providers::PhysicalInfraManager
     require_nested :EventParser
     require_nested :PhysicalServer
 
+    has_many :physical_server_profiles, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
```

Comment:
@agrare Do you think this even needed here?

Reply:
It is not, since you're already declaring this association here https://github.com/ManageIQ/manageiq/pull/21830/files#diff-8d286308b75ae7dc8a266753732b0c1709d33293518c479765e34cadf8207910R89 which is the base class, this subclass will inherit that association